packages/ckeditor5-special-characters/src/ui/charactergridview.js

```diff
@@ -128,14 +128,19 @@ export default class CharacterGridView extends View {
 				title: name
 			},
 			on: {
-				mouseover: tile.bindTemplate.to( 'mouseover' )
+				mouseover: tile.bindTemplate.to( 'mouseover' ),
+				focus: tile.bindTemplate.to( 'focus' )
 			}
 		} );
 
 		tile.on( 'mouseover', () => {
 			this.fire( 'tileHover', { name, character } );
 		} );
 
+		tile.on( 'focus', () => {
+			this.fire( 'tileFocus', { name, character } );
```

Comment:
The `tileFocus` you introduced here has to be documented in our API docs. You can check how the `tileHover` was documented and create a similar description based on that: https://github.com/ckeditor/ckeditor5/blob/38dd963e029cdff97e15fc41d8421804221f5644/packages/ckeditor5-special-characters/src/ui/charactergridview.js#L97-L105